packages/core/src/external/fhir/patient/index.ts

```diff
@@ -63,14 +63,19 @@ export const toFHIR = (patient: Pick<Patient, "id" | "data">): FHIRPatient => {
   };
 };
 
-const convertDriversLicenseToIdentifier = (
-  personalIdentifiers: PersonalIdentifier[]
-): Identifier[] => {
+export const convertPIToIdentifier = (personalIdentifiers: PersonalIdentifier[]): Identifier[] => {
   return personalIdentifiers.map(identifier => {
-    return {
-      system: driversLicenseURIs[identifier.state],
-      value: identifier.value,
-    };
+    if (identifier.type === "driversLicense") {
+      return {
+        system: driversLicenseURIs[identifier.state],
+        value: identifier.value,
+      };
+    } else {
+      return {
+        system: identifierSytemByType[identifier.type],
+        value: identifier.value,
+      };
+    }
```

Comment:
If we take the approach from [this comment](https://github.com/metriport/metriport/pull/2025/files#r1594328594), could prob reuse those functions here?

Obviously considering the shared functions living on core and not on CW. 😄 

But fine if the shapes/ are not compatible.